### ISSUE TITLE
AMD support

### ISSUE THREAD

**Issue by emoon** [sidkshatriya/rd#8]
Hi,

I know that it's stated in the readme:
> AMD Architectures are currently not supported on rd. Please use rr if you need to use AMD.

I still wonder if you have any plans on supporting AMD at some point?

**Comment by sidkshatriya** [comment]
Yes, my long term plan is to bring `rd` on par with `rr`. This will include relevant AMD support. I'm sorry I can't share a time-frame about this at this point.

Here is more info about using rr on AMD https://github.com/rr-debugger/rr/wiki/Zen in the meanwhile

Leaving this issue open for now.
